.github/workflows/publish.yml

```diff
@@ -0,0 +1,19 @@
+name: Publish
+
+on:
+  release:
+    types: [published, created, edited]
```

Comment:
Wouldn't `published` or `created` be enough? I just want to make sure that we don't run this action too often. 😁

Reply:
Agree, will change.